lib/tasks/rescan.rake

```diff
@@ -0,0 +1,6 @@
+namespace :rescan do
+  task start: :environment do
+    rescan = RescanRunner.first || RescanRunner.create
```

Comment:
I feel like more of this look could live inside the model? Now we have the same code in the controller and here.

In another project we have a public `Runner.schedule` and keep run private, so we're sure it can never be called outside of a delay queue. (Though there we have multiple runners and scheduling is a bit more involved)